notebooks/search_and_download.ipynb

```diff
@@ -39,10 +41,11 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "These will be used to authenticate your access to the API. You can either set them as environment variables in the shell or in your Python script e.g.\n",
+    "These will be used to authenticate your access to the API. You can either set them as environment variables in the shell, or in your Python script - several methods are illustrated below.\n",
     "\n",
     "**Important: When your client is initially created, you will only have access to the Archive Search. To be able to submit and download orders, please ask one of the Platform team to authorize you for performing these actions.**"
```

Comment:
Is this package intended for use by external consumers? If so, do they know how to "ask one of the Platform team"?

Reply:
Good spot, I wrote the notebook for internal use but as it's a public repo, anybody should be able to use it as an example. The best thing is to possibly insert the [Contact Us](https://www.satellitevu.com/home/#contact-us) Link on our webpage but I'm not sure whether will people will want to reach the developers directly @c-wygoda ?